iam-login-service/src/main/java/it/infn/mw/iam/api/tokens/AccessTokensController.java

```diff
@@ -76,31 +72,21 @@ public MappingJacksonValue listAccessTokens(
   @RequestMapping(method = RequestMethod.GET, value = "/{id}", produces = CONTENT_TYPE)
   public AccessToken getAccessToken(@PathVariable("id") Long id, HttpServletRequest request) {
 
-    log.debug("GET {}", request.getRequestURI());
-
-    AccessToken at = tokenService.getTokenById(id);
-
-    log.info("GET {} {}", request.getRequestURI(), HttpStatus.OK);
-    return at;
+    return tokenService.getTokenById(id);
   }
 
   @RequestMapping(method = RequestMethod.DELETE, value = "/{id}")
   public void revokeAccessToken(@PathVariable("id") Long id, HttpServletRequest request,
       HttpServletResponse response) {
 
-    log.debug("DELETE {}", request.getRequestURI());
-
     tokenService.revokeTokenById(id);
-
     response.setStatus(HttpStatus.NO_CONTENT.value());
```

Comment:
If you use the @ResponseStatus annotation you can drop the request and response parameter.
Just add @ResponseStatus(HttpStatus.NO_CONTENT) below the @RequestMapping annotation.

You can follow the same logic for the other controller methods